dev/config_fusen.yaml

```diff
@@ -39,6 +39,20 @@ flat_init_share_on_github.Rmd:
   R: R/init_share_on_github.R
   tests: tests/testthat/test-init_share_on_github.R
   vignettes: vignettes/share-on-a-github-website.Rmd
+flat_my_fun.Rmd:
```

Comment:
Is that a test of your function ? 
This is better if this does not stay in the package.  
If you think you need this example for your tests, maybe you can test it as a real unit test, where you can open the config file output.

Reply:
See previous comment.